.github/workflows/nightly-upstream-snapshot-build.yml

```diff
@@ -0,0 +1,155 @@
+on:
+  schedule:
+    - cron: "21 3 * * *"
+  workflow_dispatch:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 14
+      - uses: gradle/wrapper-validation-action@v1
+      - name: Login to GitHub Package Registry
+        run: echo ${{ secrets.CONTAINER_REGISTRY_TOKEN }} | docker login ghcr.io -u anuraaga --password-stdin
```

Comment:
We should probably make a team-wide GH account that we can use in the future instead of tying it to a single user.

Reply:
Yeah going to migrate to ECR.